docs/general_usage.md

```diff
@@ -33,6 +33,39 @@ The CLI will attempt to load credentials in the following order of priority:
 1. From environment variables, if set
 1. From the active profile
 
+## Subaccounts
+
+Instructions on how to create a Twilio subaccount can be found in the [Twilio docs](https://support.twilio.com/hc/en-us/articles/360011348693-View-and-Create-New-Twilio-Subaccounts).
+
+Once you have your subaccount SID, you can add the `--account-sid` parameter to your call to run command for subaccount.
```

Comment:
This only works for `core` commands. For other subdomains, you need to `twilio login` (or the like) with the subaccount.